pkg/deviceshifu/mockdevice/mockdevice-thermometer/mockdevice-thermometer.go

```diff
@@ -27,10 +27,10 @@ func instructionHandler(functionName string) http.HandlerFunc {
 			rand.Seed(time.Now().UnixNano())
 			min := 10
 			max := 30
-			fmt.Fprintln(w, strconv.Itoa(rand.Intn(max-min+1)+min))
+			fmt.Fprint(w, strconv.Itoa(rand.Intn(max-min+1)+min))
```

Comment:
why not Fprintln?

Reply:
Because the result of 'Fprintln' ends with \n